src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3526,4 +3526,7 @@
     <value>Yellow Gold</value>
     <comment>One of the custom color themes</comment>
   </data>
+  <data name="ClearCompleted" xml:space="preserve">
+    <value>Clear completed</value>
```

Comment:
Was the text for "clear all" removed or is it still being used?

Reply:
No. Removed.